pkg/webhooks/kafkacluster_validator.go

```diff
@@ -39,6 +39,8 @@ func (s KafkaClusterValidator) ValidateUpdate(ctx context.Context, oldObj, newOb
 	kafkaClusterOld := oldObj.(*banzaicloudv1beta1.KafkaCluster)
 	kafkaClusterNew := newObj.(*banzaicloudv1beta1.KafkaCluster)
 	log := s.Log.WithValues("name", kafkaClusterNew.GetName(), "namespace", kafkaClusterNew.GetNamespace())
+
+	// check storage removal
```

Comment:
Suggestion: This comment in unnecessary. 

Something that I learned from reading a book called [Clean Code](https://books.google.com/books/about/Clean_Code.html?id=hjEFCAAAQBAJ&source=kp_book_description)(which I highly recommend) is that adding comment that is not needed to the source code is actually considered a bad coding behavior because it requires the audience to spend time on reading the comment. 

And I think this comment falls into the "unnecessary comment" category because the next line calls a function named `checkBrokerStorageRemoval`, which is self-explanatory enough